sdk/communication/communication-administration/README.md

```diff
@@ -134,9 +134,19 @@ await client.deleteUser(user);
 #### Creating an instance of PhoneNumberAdministrationClient
 
 ```typescript
-import { CommunicationIdentityClient } from "@azure/communication-administration";
+import { PhoneNumberAdministrationClient } from "@azure/communication-administration";
 
-const client = new CommunicationIdentityClient(CONNECTION_STRING);
+const client = new PhoneNumberAdministrationClient(CONNECTION_STRING);
+```
+
+#### Creating an instance of PhoneNumberAdministrationClient with TokenCredential
```

Comment:
Is saying TokenCredential enough for users to know that they can authenticate using managed identity (Azure Active Directory Authentication)?

Reply:
Yes that is fair I think we can make this change to be more consistent with the other platforms. I will update the change log as well.